action.yml

```diff
@@ -14,7 +14,7 @@ runs:
         if [ "${VERSION}" = "latest" ]; then
           DOWNLOAD_URL=$(curl -sS https://api.github.com/repos/kayac/ecspresso/releases | jq -r '[.[]|select(.tag_name < "v1.99")|select(.prerelease == false)][0].assets[].browser_download_url|select(match("linux.amd64."))')
         else
-          DOWNLOAD_URL=$(curl -sS https://api.github.com/repos/kayac/ecspresso/releases/tags/${VERSION}|jq -r '.assets[].browser_download_url|select(match("linux.amd64."))')
+          DOWNLOAD_URL=https://github.com/kayac/ecspresso/releases/download/v${VERSION}/ecspresso_${VERSION}_linux_amd64.tar.gz
```

Comment:
`$VERSION` has a prefix `v` because that is based on git tags.
So this URL seems to be expanded as `https://github.com/kayac/ecspresso/releases/download/vv2.0.0/ecspresso_v2.0.0_linux_amd64.tar.gz`.